File: FE/shopapp-angular/src/app/responses/order/order.response.ts
export interface OrderResponse {
    id: number;
    fullname: string;
    phone_number: string;
    email: string;
    address: string;
    note: string;
    user_id: number;
    total_money: number;
    order_date: Date;
    status: string;
    shipping_method: string;
    payment_method: string;
    shipping_date: Date;
    tracking_number: string;
    shipping_address: string;
    created_at: number[];
    updated_at: number[];
}

File: BE/shopapp/src/main/java/com/project/shopapp/DTOs/responses/OrderResponse.java
package com.project.shopapp.DTOs.responses;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.*;

import java.math.BigDecimal;
import java.time.LocalDate;
import java.util.Date;

//@Data
@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class OrderResponse extends BaseResponse{
    private long id;

    @JsonProperty("fullname")
    private String fullName;

    @JsonProperty("phone_number")
    private String phoneNumber;

    private String email;
    private String address;
    private String note;

    @JsonProperty("user_id")
    private int userId;

    @JsonProperty("total_money")
    private BigDecimal totalMoney;

    @JsonProperty("order_date")
    private Date orderDate;

    private String status;

    @JsonProperty("shipping_method")
    private String shippingMethod;

    @JsonProperty("payment_method")
    private String paymentMethod;

    @JsonProperty("shipping_date")
    private LocalDate shippingDate;

    @JsonProperty("tracking_number")
    private String trackingNumber;

    @JsonProperty("shipping_address")
    private String shippingAddress;

}
